hackathon-website/src/Components/ImagineMare/ImagineMare.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "./ImagineMare.css";
+
+const ImagineMare = () => (
```

Comment:
Pune te rog un nume mai sugestiv si clar pentru componenta, si in engleza. 
Spre exemplu LogoHermesLarge, LogoHermesBig, LogoHermes etc.